src/test/kotlin/uk/gov/dwp/dataworks/controllers/UserContainerControllerTest.kt

```diff
@@ -48,12 +52,21 @@ class UserContainerControllerTest {
     @Test
     fun `200 returned with well formed request`() {
         mvc.perform(MockMvcRequestBuilders.post("/deployusercontainers")
-                .content("{\"ecsClusterName\": \"Test Cluster Name\","
-                        + " \"userName\": \"Test User Name\"," +
-                        "\"emrClusterHostName\": \"Test EMR Host Name\"," +
-                        "\"albName\": \"Test alb Name\"" +
-                        "}")
+                .content("{\"userName\" : \"test_user\"}")
                 .header("content-type", "application/json"))
                 .andExpect(MockMvcResultMatchers.status().isOk)
     }
+
+    @Test (expected = Exception::class)
```

Comment:
Can we check for a more specific Exception class than just `Exception` as this will catch for _any_ failure.